src/pages/home/HeaderView.js

```diff
@@ -96,6 +99,15 @@ const HeaderView = (props) => {
                             </View>
                         </Pressable>
                         <View style={[styles.reportOptions, styles.flexRow]}>
+                            <Pressable
+                                onPress={() => props.onVideoChatMenuButtonClicked()}
+                                style={[styles.touchableButtonImage, styles.mr0]}
+                            >
+                                <Icon
+                                    src={Trashcan}
+                                    fill={props.report.isPinned ? themeColors.heading : themeColors.icon}
```

Comment:
I'm not sure whether the report is pinned should affect the icon color here. Ideally it would be `themeColors.heading` when there's an active meeting in that room, or `themeColors.icon` otherwise. But since that would require more complex logic I think it should just always be `themeColors.icon` (maybe allowing for some hover styles too if you want).